### PR TITLE
Psp 525 update yml default support

### PR DESCRIPTION
**Pull request by mozrokafor**
# jira issue id
https://mozilla-hub.atlassian.net/browse/PSP-525

# Checklist

- [ ] l10n changes have been submitted to the l10n repository, if any.
- [ ] All acceptance criteria are met.
- [ ] I've added or updated relevant docs in the docs/ directory.
- [ ] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/frontend/src/styles/tokens.scss`).
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -36,9 +36,11 @@ jobs:
...
 
       - name: Run Playwright tests
-        run: npm run test:${{ inputs.environment }}
-        env:
-          E2E_TEST_ENV: ${{ inputs.environment }}
+        run: |
+          commandenv=":${{ inputs.environment != null && inputs.environment || 'stage' }}"
+          npm run test:$commandenv
+        env:          
+          E2E_TEST_ENV: ${{ inputs.environment != null && inputs.environment || 'stage' }}
```
Comment (Vinnl) [inline .github/workflows/playwright.yml:43]: If I'm reading it correctly, I think this line isn't used?

```suggestion
```

File: e2e-tests/e2eTestUtils/helpers.ts [hunk 2]
```diff
@@ -95,10 +95,10 @@ export const generateRandomEmail = async () => {
 
 export const setEnvVariables = async (email: string) => {  
   // set env variables
-  // stage will currently be the default
+  // stage will currently be the default
```
Comment (Vinnl) [inline e2e-tests/e2eTestUtils/helpers.ts:98]: Haha nice, you got the comment in before the code that it describes :grin:

File: e2e-tests/specs/relay-e2e.spec.ts [hunk 3]
```diff
@@ -27,7 +27,8 @@ test.describe('Relay e2e function email forwarding', () => {
     })
 })
 
-test.describe('Relay e2e auth flows', () => {
+test.fixme('Relay e2e auth flows', () => {
+  // pricing table is being updated -- will update this test afterwords
```
Comment (Vinnl) [inline e2e-tests/specs/relay-e2e.spec.ts:31]: Thanks for the comment clarifying the `.fixme`!

**Review by Vinnl — APPROVED** [review]

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (mozrokafor) [reply .github/workflows/playwright.yml:43]: its used, essentially what this line does in yml is use inputs.environment if its provided (not null), else if null use 'stage'

Comment (Vinnl) [reply .github/workflows/playwright.yml:43]: Ah got it, the env var is used in the test itself, not in the command. I assume it should be posible to change it to the following?

```
        run: npm run test:$E2E_TEST_ENV
        env:          
          E2E_TEST_ENV: ${{ inputs.environment != null && inputs.environment || 'stage' }}
```

**Pull request merged**